fed/mosaic-ns3/src/main/java/org/eclipse/mosaic/fed/ns3/ambassador/Ns3Ambassador.java

```diff
@@ -54,10 +54,9 @@ public FederateExecutor createFederateExecutor(String host, int port, OperatingS
             case WINDOWS:
             case UNKNOWN:
             default:
-                log.error("Operating system not supported");
-                break;
+                log.error("Operating system not supported by ns3");
+                throw new RuntimeException("Operating system not supported by ns3");
```

Comment:
Makes sense. For cases where we want to use ns-3 on windows by starting it separately (e.g. in WSL), we now need to disable "start" via "runtime.json".